src/utils/nameUtils.ts

```diff
@@ -137,6 +137,9 @@ export function normalizeName(
   nameType: NameType,
   shouldGuard?: boolean
 ): string {
+  if (name.startsWith("$DO_NOT_NORMALIZE$")) {
+    return name.replace("$DO_NOT_NORMALIZE$", "");
```

Comment:
Can you please file an issue in M4 to see if we can get an overridenName property in the language.default object

Reply:
I do not have access to create issue in M4. But, I have sent an email detailing the issue.